apps/crn-server/serverless.ts

```diff
@@ -846,10 +846,10 @@ const serverlessConfig: AWS = {
             pattern: {
               source: [eventBusSourceSquidex],
               'detail-type': [
-                'GroupsPublished',
-                'GroupsUpdated',
-                'GroupsUnpublished',
-                'GroupsDeleted',
+                'InterestGroupsPublished',
```

Comment:
I think this will break the hook as the name is still "groups" in the schema although its label is "Interest Groups"

<img width="1440" alt="interest-groups-name" src="https://github.com/yldio/asap-hub/assets/16595804/b35e4056-539c-4fea-acd2-fd45a267d019">

So when you update a interest group and go check the webhook call in `Rules` -> `Logs`, we can see that the name of the event is still `GroupsUpdated`

<img width="1439" alt="interest-groups-hook" src="https://github.com/yldio/asap-hub/assets/16595804/3fb02e07-85a1-45a5-84dc-21bddd65a0f0">

I believe we would need to change the name https://github.com/yldio/asap-hub/blob/CRN-1228-refactor-all-code-components-from-groups-to-interest-groups/packages/squidex/schema/crn/schemas/interest-groups.json#L358 to `interest-groups` as well. But it would be nice to check this flow after updating the name to see if it works :)